fastlane_bot/helpers/txhelpers.py

```diff
@@ -457,47 +464,21 @@ def build_transaction_with_gas(
 
         returns: the transaction to be submitted to the blockchain
         """
+        value = src_amt if (src_address == self.ConfigObj.NATIVE_GAS_TOKEN_ADDRESS and self.ConfigObj.SELF_FUND) else None
+        transaction = self.build_transaction_generic(
+                                                    self.construct_contract_function,
+                                                    routes,
+                                                    src_amt,
+                                                    src_address,
+                                                    flashloan_struct,
+                                                    gas_price=gas_price,
+                                                    max_priority_fee=max_priority_fee,
+                                                    nonce=nonce,
+                                                    value=value
```

Comment:
Minor: the indentation of all of these parameters should be reduced by a lot.

Reply:
Updated